all-repos.yaml

```diff
@@ -226,3 +226,4 @@
 - https://gitlab.com/adam-moss/pre-commit-ssh-git-signing-key
 - https://github.com/charliermarsh/ruff-pre-commit
 - https://github.com/mrtazz/checkmake
+- https://github.com/jshwi/docsig
```

Comment:
why does this use `require_serial: true` ?

Reply:
Hi @asottile,
Thank you for checking this
I'm new to `pre-commit`, I sort of inherited from a hook I was learning from
Come to think of it the hooks I've seen since don't have that, removed